node/src/daemon/setup_reporter.rs

```diff
@@ -1283,7 +1283,7 @@ mod tests {
             ("ip", "4.3.2.1", Set),
             ("log-level", "warn", Default),
             ("mapping-protocol", "", Blank),
-            ("min-hops", "3", Configured), // TODO: GH-698: This should be changed to Default after this card is played
+            ("min-hops", "", Blank),
```

Comment:
It will become `Default` if it is provided by the Clap. Now, we're not using Clap to provide the default value, instead we're using Multi Config and Database for identifying the value.

Priority Order is `Multi Config > Database`.

For a newly created Databases, the default count of 3 will be picked.

Due to that reason, it should stay `Blank`.

Reply:
Done 👍